### DISCUSSION TITLE
模型参数知多少 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#443] [General]
# 模型参数知多少 - Miao Yu | 于淼

https://yufree.cn/cn/2023/11/27/number-of-model-parameters/

**Comment by earfanfan** [comment]
原来新冠以后是可以完全恢复的啊，我已经连着三年白细胞数和中性粒细胞数低于正常范围下限了，上网搜是说这算是免疫力下降了。

**Reply by yufree** [reply]
不一定，正常范围可能跟你基线水平不一样，需要跟历年体检结果去比较。我恢复的静息心率还是低于60，也就是低于正常范围的下限。

**Reply by earfanfan** [reply]
话说三水你判断正常体温范围的上限是多少？我今早开始头晕恶心嗓子疼，但是体温只到了37.3度（测的额头），除了让我感到整个脑袋尤其是脸颊烫得难受以外，整体确实也算正常。这种踩在临界值点的温度，让我偶然怀疑是不是新冠三年以后，医院或者研究机构应该用一批新样本重新定义正常范围了。

**Reply by yufree** [reply]
我判断不了，你要拿不准，每天固定时间用固定测量方式测同一个部位的体温一个月就能得到你自己的正常范围了。
